src/model/user.h

```diff
@@ -210,6 +210,8 @@ class TOGGL_INTERNAL_EXPORT User : public BaseModel {
         const std::string &password,
         std::string *result);
 
+    bool timeline_ui_flag = true;
```

Comment:
changing this to `false` doesn't hide the tab switcher

Reply:
It does, but you have to make full rebuild for this. I don't know why, but changes in user.h are noticed by VS compiler.

Reply:
+